server/conn.go

```diff
@@ -530,6 +532,17 @@ func (cc *clientConn) handleLoadData(loadDataInfo *executor.LoadDataInfo) error
 
 func (cc *clientConn) handleQuery(sql string) (err error) {
 	startTs := time.Now()
+	defer func() {
+		// Add metrics
+		t := time.Since(startTs)
+		queryHistgram.Observe(float64(t))
```

Comment:
But most of out query will be handled in less than 30ms. I will rename it to handle_query_duration_nanoseconds.